compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -281,6 +281,14 @@ void addPackPeelBasedPassPipeline(OpPassManager &funcPassManager,
   funcPassManager.addPass(createHoistStaticallyBoundAllocationsPass());
   funcPassManager.addPass(createCanonicalizerPass());
 
+  // Lower to UKernels.
+  {
+    AMDAIELowerToUKernelsOptions options;
+    options.passPipeline = AIEPassPipeline::PackPeelPipeline;
```

Comment:
How is the pass pipeline used in the ukernel lowering? I've tried partially removing it in https://github.com/nod-ai/iree-amd-aie/pull/345 and it works, although I haven't tried removing it from AMDAIELowerToUKernelsOptions

Reply:
Yes, you're right - it's not needed. I forgot to remove it - thanks for pointing this out. I will do that in this PR.

Reply:
Oh. I didn't see this and have thus removed it in this PR.

Reply:
Let me know what would you prefer - i) merging just this PR or ii) the other PR followed by this.

Reply:
(i) makes sense, I've closed the other PR now